### ISSUE TITLE
Add watermark/overlay to all videos

### ISSUE THREAD

**Issue by Mantene** [HaveAGitGat/Tdarr_Plugins#729]
Is it possible to get a plugin, or even just instructions if it is already possible, that adds an overlay/watermark to the first 15 seconds of each video? Like a small png file or something, and position it in the lower right corner?

**Comment by HaveAGitGat** [comment]
Yes you can use this `Tdarr_Plugin_00td_action_handbrake_ffmpeg_custom`

Set ffmpeg.

Then arguments:

`<io> -i C;/logo.png -filter_complex \"[0:v][1:v]overlay=10:10:enable=between(t\,0\,15)" `

That should work
